### ISSUE TITLE
Error saving yaml file with hashed passwords

### ISSUE THREAD

**Issue by oparex** [apotdevin/thunderhub#153]
**Describe the problem/bug**
On startup, I am presented with the users list. Clicking on the username I wish to use does nothing.

**Your environment**
Ubuntu 20
$ node -v -> v10.19.0
$ npm -v -> 6.14.4
* Version of ThunderHub:
0.9.14
* Deployment method: 
npm run build

I am using lndDir parameter in yaml.
There is a problem when saving the hashed password in yaml config. With silly log level, no additional information is provided.

$ npm start -- -p 4000

> thunderhub@0.9.14 start /home/peter/node/src/thunderhub
> next start "-p" "4000"

info  - Loaded env from /home/peter/node/src/thunderhub/.env.local
info  - Loaded env from /home/peter/node/src/thunderhub/.env
ready - started server on http://localhost:4000
2020-10-07 06:13:16 verbose [THUB]: No cookie path provided
2020-10-07 06:13:17 verbose [THUB]: No macaroon path provided
2020-10-07 06:13:18 info [THUB]: Saving new yaml file with hashed passwords
2020-10-07 06:13:18 error [THUB]: Error saving yaml file with hashed passwords. Passwords are still in cleartext on your server.
2020-10-07 06:13:18 info [THUB]: Server accounts that will be available: peter
2020-10-07 06:13:18 silly [THUB]: Authentication cookie failed

**Comment by apotdevin** [comment]
Hey! Seems like there was some error saving the hashed file. Does ThunderHub have the correct write permissions in the yaml config file folder?

**Comment by oparex** [comment]
It is running without the error now. I ran the whole thing without masterPasword when the above error was in the logs, then I have added masterPassword and it worked, no more error when the home page loads. There are also hashes of passwords in the config file now present.

Thunderhub still does not work for me. On the home page, I see my username and a login button but nothing happens when I click the button. This is another issue I guess.
